### PR TITLE
Fix audit and remediation for CIS-1.9 master 1.1.13/1.1.14

### PR DESCRIPTION
**Pull request by andypitcher**
Parent:
-  https://github.com/aquasecurity/kube-bench/issues/1644

### CODE REVIEW

File: cfg/cis-1.9/master.yaml [hunk 1]
```diff
@@ -189,7 +189,7 @@ groups:
       - id: 1.1.13
         text: "Ensure that the default administrative credential file permissions are set to 600 (Automated)"
         audit: |
-         for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c \"permissions=%a %n\" $adminconf; fi; done
+         for adminconf in /etc/kubernetes/{admin.conf,super-admin.conf}; do if test -e $adminconf; then stat -c "permissions=%a %n" $adminconf; fi; done
```
Comment (mozillazg) [inline cfg/cis-1.9/master.yaml:192]: alpine shell doesn't support this syntax: `{admin.conf,super-admin.conf}`

Comment (andypitcher) [reply cfg/cis-1.9/master.yaml]: @mozillazg I changed that to match what you proposed in the original issue.

**Review by mozillazg — APPROVED** [review]
LGTM, Thanks for your contribution!

**Comment by mozillazg** [comment]
@chen-keinan ping~

**Comment by afdesk** [comment]
hi @mozillazg! this PR is ready for merge, right?

**Review by afdesk — APPROVED** [review]
LGTM

**Comment by mozillazg** [comment]
> hi @mozillazg! this PR is ready for merge, right?

@afdesk Yes, it is ready for merge.

**Comment by afdesk** [comment]
> > hi @mozillazg! this PR is ready for merge, right?
> 
> @afdesk Yes, it is ready for merge.

great!

**Pull request merged**